frontend/src/lib/components/LemonModal/LemonModal.scss

```diff
@@ -1,57 +1,100 @@
-.LemonModal {
-    .ant-modal-close-x {
-        position: absolute;
-        top: 0.75rem;
-        right: 0.75em;
-        height: auto;
-        width: auto;
-    }
-    .ant-modal-header,
-    .ant-modal-body,
-    .ant-modal-footer {
-        padding: 1rem;
-    }
-    .ant-modal-header,
-    .ant-modal-footer {
-        display: flex;
-        align-items: center;
-        line-height: 1.5rem;
-        border-color: var(--border);
-    }
+.LemonModal__overlay {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+
+    transition: background-color 250ms ease-out, backdrop-filter 250ms ease-out;
+    z-index: var(--z-modal);
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
 
-    .ant-modal-header {
-        border-bottom: none;
+    &.ReactModal__Overlay--after-open {
+        background-color: rgba(0, 0, 0, 0.4);
+        backdrop-filter: blur(5px);
```

Comment:
I suggest putting this blur strength into a CSS variable, so that this is more explicitly part of the design system too (and sync with @lottiecoxon @clarkus so that designs include the backdrop blur too).

Reply:
Agreed. I abstracted it now and matched the Design System - 10px. To me it feels too strong but happy to defer to @clarkus / @lottiecoxon for input there

Reply:
Actually reverted to 5px as that matches the design closer. I think it's due to blur being based on resolution. Browsers cleverly scale up to "retina" so 5px is 10px with a higher DPI. _I think..._

|Figma (10px)|Code(5px)|Code(10px)|
|----|----|----|
|<img width="281" alt="Screenshot 2022-08-09 at 17 00 01" src="https://user-images.githubusercontent.com/2536520/183685434-1d078f3f-ada2-4d25-a925-374e948bb55f.png">|<img width="357" alt="Screenshot 2022-08-09 at 17 00 06" src="https://user-images.githubusercontent.com/2536520/183685437-bd093a5a-3449-49e7-8237-f768d4f947af.png">|<img width="403" alt="Screenshot 2022-08-09 at 17 00 59" src="https://user-images.githubusercontent.com/2536520/183685601-fff31bc1-71f0-455c-9538-047d5055eae1.png">|